CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
+# 2.12.0

```

Comment:
Looks like we need to fix the release notes and merge the fix into develop first. Close this PR and create a branch with develop and master merged together, then PR that into develop.